admin/js/boldgrid-backup-admin.js

```diff
@@ -112,14 +114,77 @@ BoldGrid.Backup = function( $ ) {
 		} );
 	};
 
+	/**
+	 * Auto Update Themes
+	 *
+	 * Add functionality for WP >= 5.5 theme update links.\
+	 *
+	 * @since SINCEVERSION
+	 */
+	self.autoUpdateThemes = function( link ) {
+		var data = {
+			action: 'boldgrid_backup_auto_update',
+			_wpnonce: BoldGridBackupAdmin.auto_update_nonce,
+			data: link.dataset
+		};
+		$.post( ajaxurl, data, function( response ) {
+			if ( JSON.parse( response ).settings_updated ) {
+				location.reload();
+			}
+		} );
+	};
+
+	/**
+	 * Auto Update Links
+	 *
+	 * Adds functionality for WP >= 5.5 update links.
+	 *
+	 * @since SINCEVERSION
+	 */
+	self.autoUpdateLinks = function() {
```

Comment:
What does `autoUpdateLinks` mean? Does it mean listen to clicks of auto update links, add the links, modify the links?